MODULE.bazel

```diff
@@ -6,3 +6,7 @@ module(
 
 bazel_dep(name = "platforms", version = "0.0.7")
 bazel_dep(name = "rules_cc", version = "0.0.8")
+bazel_dep(name = "rules_python", version = "0.27.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+use_repo(python, "pythons_hub")
```

Comment:
The `@pythons_hub` is an internal detail of rules_python. It's basically just a registry so that it can auto-register the toolchains and make pip integration easier.

Reply:
I feared as much. Hence why I really wanted you to review. :)